### ISSUE TITLE
Button Text is out of range

### ISSUE THREAD

**Issue by hxbb00** [picoe/Eto#1722]
as title

## Actual Behavior

![Snipaste_2020-06-14_10-25-48](https://user-images.githubusercontent.com/14104435/84583300-aac2b280-ae29-11ea-9a63-1cf104374337.png)

  - Version:
  - Platform(s): Gtk3
  - Operating System(s): Windows 10,

**Comment by cwensley** [comment]
Hey @hxbb00,

Do you have more information on the problem?  e.g. a reproduction of what you are doing?  Is that the main menu, or is that a button?  I can't really tell..

**Comment by hxbb00** [comment]
just button

**Comment by cwensley** [comment]
Hey @hxbb00, a sample of the code you are using would help, and would answer these questions:

Are you setting its width to something less than the required width?  Are you expecting the text to be cut off or are you expecting the button to be wider?  What container is it in?

**Comment by hxbb00** [comment]
thank  you @cwensley, i expecting the text to be cut off,
here the code
```csharp
using Eto.Drawing;
using Eto.Forms;
using System;

namespace TkMapApp
{
    public class MyForm : Form
    {
        public MyForm()
        {
            Title = "My Cross-Platform App";
            ClientSize = new Size(200, 200);

            var panel = new Panel();
            StackLayout stackLayout = new StackLayout();
            Content = stackLayout;

            panel.Width = 60;
            panel.Content = new Button
            {
                Text = "Hello World2, out of range!"
            };

            stackLayout.Items.Add(new StackLayoutItem(panel));
        }

        [STAThread]
        static void Main()
        {
            new Application(new Eto.GtkSharp.Platform()).Run(new MyForm());
        }
    }
}
```

**Comment by cwensley** [comment]
Thanks @hxbb00, the code example and expected behaviour really helps narrow things down.

**Comment by cwensley** [comment]
Thanks again for reporting the issue!  This should be fixed in the develop branch and will be included in the 2.5.3 release.

Note that on Gtk+3 you cannot simply clip the contents, but will now have an ellipsis at the end.
